var/spack/repos/builtin/packages/kripke/package.py

```diff
@@ -4,43 +4,95 @@
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
 from spack.package import *
+import os
 
 
-class Kripke(CMakePackage):
+class Kripke(CMakePackage, CudaPackage, ROCmPackage):
     """Kripke is a simple, scalable, 3D Sn deterministic particle
     transport proxy/mini app.
     """
 
     homepage = "https://computing.llnl.gov/projects/co-design/kripke"
     git = "https://github.com/LLNL/Kripke.git"
 
-    tags = ["proxy-app"]
-    version("1.2.4", submodules=True, tag="v1.2.4")
+    tags = ["SN-neutronics proxy-app"]
```

Comment:
Is this supposed to be two separate tags?  If so, should be a comma-separated list of two strings.

I suspect we don't support spaces within a tag.